routes/api.php

```diff
@@ -2,6 +2,7 @@
 
 Route::get('validations')->uses('Api\ValidationsController@view')->name('api.validations');
 Route::get('metar/{airportIcao}')->uses('Site\MetarController@get')->name('api.metar');
+Route::get('/cts/bookings')->uses('Api\CtsController@getBookings')->name('api.cts.bookings');
```

Comment:
General feedback here on a public endpoint: we'd like to apply some rate limits here to ensure we don't get undue load on our database.